headers_test.go

```diff
@@ -251,6 +251,13 @@ func TestProtectedHeader_UnmarshalCBOR(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "int map key too large",
```

Comment:
We also need this test for unprotected header.

Reply:
Done!